data/Census-State/README.md

```diff
@@ -0,0 +1 @@
+# Census-State
```

Comment:
Similar to FAA-Airports, add a short paragraph description and download instructions for Cenus-State